01_maj_pattern_book.md

```diff
@@ -23,464 +23,3 @@ these origins were unknown to us.  If we have erred in our attributions, please
 ![](./media/majbanner.jpeg){width=100%} 
 
 
-# Hello Madison Club Passers and Friends!
```

Comment:
This is chapter 2 in the pdf printout.  Makes more sense to me to put it in its own numbered markdown file.  This means the subsequent mark down files need to be incremented, which is also nice the file numbers now match the chapter number in the pdf printout.